### DISCUSSION TITLE
Nexus Tower Music

### DISCUSSION THREAD

**Discussion by jadebenn** [DarkflameUniverse/DarkflameServer#1249] [General]
Is Nexus Tower music supposed to be working? Not hearing anything when I load it on my client.

**Comment by aronwk-aaron** [comment]
audio is a weird and complex system, sometimes audio break. just restart your client and it should work

**Reply by jadebenn** [reply]
It might be general client incompatibility, yeah. I know the sound system isn't aging too gracefully. Just trying to get confirmation that the issue was on my end,   not the server. Sounds like it is.
